website/source/docs/providers/rancher/r/environment.html.md

```diff
@@ -28,6 +40,7 @@ The following arguments are supported:
 * `name` - (Required) The name of the environment.
 * `description` - (Optional) An environment description.
 * `orchestration` - (Optional) Must be one of **cattle**, **swarm**, **mesos** or **kubernetes**. Defaults to **cattle**.
+* `member` - (Optional) Members to add to the environment. Each member has `external_id`, `external_id_type` and `role` values.
```

Comment:
Would be helpful to add a section before the "Attributes Reference" with more details on what each of the parameters accepted inside the `member` block is and what their type is.